contrib/de_DE/LC_MESSAGES/gimp.po

```diff
@@ -26,14 +26,18 @@ msgstr ""
 msgid "Export Image %i/%i"
 msgstr "Exportiere Bild %i/%i"
 
-#: gimp.lua:194
+#: gimp.lua:221
+msgid "Unable to move edited file into collection.  Leaving it as %s"
+msgstr "Kann nicht bearbeitete Datei in Sammlung zu bewegen. Wenn sie unverändert %s"
```

Comment:
Better:
"Die bearbeitete Datei kann nicht in die Sammlung aufgenommen werden. %s bleibt unverändert in der Sammlung"